frontend/projects/upgrade/src/app/features/dashboard/profile/components/metrics/metrics.component.html

```diff
@@ -32,64 +43,58 @@
-                    <mat-icon *ngIf="permissions.metrics.delete && keyEditMode" class="icon" (click)="deleteNode(node)"
-                      >delete_outline</mat-icon
-                    >
-                  </li>
-                </mat-tree-node>
-
-                <!-- With Children -->
-                <mat-nested-tree-node *matTreeNodeDef="let node; when: hasNestedChild">
-                  <li>
-                    <button mat-icon-button matTreeNodeToggle>
-                      <mat-icon class="mat-icon-rtl-mirror">
-                        {{ nestedTreeControl.isExpanded(node) ? 'expand_more' : 'chevron_right' }}
-                      </mat-icon>
-                    </button>
-                    <span class="ft-14-600">
-                      {{ node.key }}
-                    </span>
-                    <mat-icon *ngIf="permissions.metrics.delete && keyEditMode" class="icon" (click)="deleteNode(node)"
-                      >delete_outline</mat-icon
-                    >
-                    <ul [class.node-tree-invisible]="!nestedTreeControl.isExpanded(node)">
-                      <ng-container matTreeNodeOutlet></ng-container>
-                    </ul>
-                  </li>
-                </mat-nested-tree-node>
-              </mat-tree>
-            </ng-template>
+          <td class="ft-12-600" mat-cell *matCellDef="let element">
+            <!-- {{ element.key }} -->
+            <mat-tree [dataSource]="[element]" [treeControl]="nestedTreeControl" class="example-tree">
+              <!-- This is the tree node template for leaf nodes -->
+              <mat-tree-node *matTreeNodeDef="let node; let index = index">
+                <button mat-icon-button disabled class="button-container">
+                  <!-- <mat-icon class="mat-icon-rtl-mirror"> chevron_right </mat-icon> -->
+                </button>
+                {{ node.key }}
+                <mat-icon *ngIf="permissions.metrics.delete && keyEditMode" class="icon" (click)="deleteNode(node)">
+                  delete_outline
+                </mat-icon>
+              </mat-tree-node>
+              <!-- This is the tree node template for expandable nodes -->
+              <mat-nested-tree-node *matTreeNodeDef="let node; when: hasNestedChild">
+                <div class="mat-tree-node">
+                  <button mat-icon-button matTreeNodeToggle [attr.aria-label]="'Toggle ' + node.key">
+                    <mat-icon class="mat-icon-rtl-mirror">
+                      {{ nestedTreeControl.isExpanded(node) ? 'expand_more' : 'chevron_right' }}
+                    </mat-icon>
+                  </button>
+                  {{ node.key }}
+                  <mat-icon *ngIf="permissions.metrics.delete && keyEditMode" class="icon" (click)="deleteNode(node)"
+                    >delete_outline</mat-icon
+                  >
+                </div>
+                <div [class.example-tree-invisible]="!nestedTreeControl.isExpanded(node)" role="group">
```

Comment:
replace the name of the class `.example-tree` with the relevant name in the HTML and CSS file.